### ISSUE TITLE
EasyFSL on an image classification problem with my own dataset

### ISSUE THREAD

**Issue by wongshujun** [sicara/easy-few-shot-learning#77]
I am a complete beginner in python coding and I am tasked with an image classification problem. 
I have a dataset of 25000 images with 2 classes (50-50). 1 class being photos of corrosion and the other with photos of non-corrosion.

How do I change the code in my_first_few_shot_classifier.ipynb so that I can upload my own dataset from my local drive as well as split the dataset into training and test set.

Thank you!

**Comment by ebennequin** [comment]
This code is meant to be used for Few-Shot Image Classification, which seems to be very far from your setting. I suggest you follow tutorials on image classification with custom datasets, such as [this one](https://towardsdatascience.com/custom-dataset-in-pytorch-part-1-images-2df3152895).

**Comment by wongshujun** [comment]
Is there a way I can load my own custom dataset that will be compatible with the current FSL code because my supervisor insists on the area of FSL. Thank you!

**Comment by ebennequin** [comment]
Sure! This question is answered in issue #17. Hope that helps!
